File: python/main4.py
def dangerous(value1: int, value2: int) -> int:
    return value1 // value2
def foo(array: list[int], counter: int) -> int:
    if counter == 0:
        return dangerous(array[0], counter)
    return foo(array, counter - 1)
def main() -> None:
    array: list[int] = [0] * 1000
    print("The result is %i" % foo(array, 6))
if __name__ == "__main__":
    main()

File: java/Main4.java
public class Main4 {
    static int dangerous(int value1, int value2){ return value1 / value2; }
    static int foo(int[] array, int counter){
        if(counter==0) return dangerous(array[0], counter);
        return foo(array, counter-1);
    }
    public static void main(String[] args){
        int[] array = new int[1000];
        System.out.printf("The result is %d%n", foo(array, 6));
    }
}

